### PR TITLE
Feat: 회원 기능 구현을 위한 엔티티-DTO간 매핑 레포지토리 생성 #83

### PR DESCRIPTION
**Pull request by speter6501**
요청하신 작업 날립니다

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```
Comment (DokaDev) [inline gradle/wrapper/gradle-wrapper.properties:3]: 이건 어떤거때문에 변경사항 넣으신거죠?

**Review by DokaDev — CHANGES_REQUESTED** [review]
코멘트확인해주세요

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (speter6501) [reply gradle/wrapper/gradle-wrapper.properties:3]: 인텔리제이 안에서 로드가 안되서 넣었습니다

Comment (DokaDev) [reply gradle/wrapper/gradle-wrapper.properties:3]: 저게 8.8일때는 로드가 되고 8.5에서는 로드가 안된다는 말씀이세요?

Comment (DokaDev) [reply gradle/wrapper/gradle-wrapper.properties:3]: 지금저거 파일추가가 아니라 변경이력같은데요

Comment (speter6501) [reply gradle/wrapper/gradle-wrapper.properties:3]: 그게 아니라 8.8일때 로드가 안되고 8.5일때 로드가 되서 인텔리제이 설정 다다시 해줘야 되서 변경했습니다

Comment (DokaDev) [reply gradle/wrapper/gradle-wrapper.properties:3]: 로드 안되는 이슈 혹시 캡처해서 올려주실수있나요? 저걸 곧바로 개발버전에 업데이트하기엔 위험부담이 좀 있어보이네요.

Comment (DokaDev) [reply gradle/wrapper/gradle-wrapper.properties:3]: 어떤 작업을 요청하는 상황에서 어떠한 이유로 어떤 형태로 로드가 되지 않는지 구체적으로 말씀부탁드립니다.

Comment (speter6501) [reply gradle/wrapper/gradle-wrapper.properties:3]: 오후 6:23:57: Executing ':classes'...


FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'rvcom'.
> Multiple build operations failed.
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\8de7cba9eaf3f8277a748b1a97834f88-006bdbf7-983e-4b71-a777-d0cae873f077) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\8de7cba9eaf3f8277a748b1a97834f88)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1ccbe0288edd62f65586af9879c65973-0d27e140-d63c-4c2f-9bd7-6ebcf454f191) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1ccbe0288edd62f65586af9879c65973)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\cb4e8f2b118d47f35dd7e49671d4b502-460588fd-5219-43e5-888a-4015f68e947c) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\cb4e8f2b118d47f35dd7e49671d4b502)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\a0d121f6bf0a0cd1ab39dd6508a57d7e-9a300cef-f2d9-4cf6-a112-598de9f17a97) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\a0d121f6bf0a0cd1ab39dd6508a57d7e)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\59c8832e15832b6df264a2872e671865-c6fde3de-4dbc-44da-ac9e-e473d692bcf4) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\59c8832e15832b6df264a2872e671865)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\12737c2f4a859b2f660c843c79d45a86-07faf332-d9dd-4535-a3dd-71a93e78888b) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\12737c2f4a859b2f660c843c79d45a86)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\28bed1eec1ad409b80a9bdcf26268253-82bee0f8-c8ed-4a40-ba48-1e027e4ff52c) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\28bed1eec1ad409b80a9bdcf26268253)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1ae2b520ba5c3afbc50058c5642ff12b-5ccb2758-0c79-4e87-8b09-103300f36438) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1ae2b520ba5c3afbc50058c5642ff12b)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\53938ff7056193752b70a38472c61efe-943788c1-3281-48fd-bdb5-2e8a610f36a1) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\53938ff7056193752b70a38472c61efe)
      Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1fbff5c01f8d08c16d1419f8d14eabd6-6ba1bb7e-2374-419d-b796-026621206032) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1fbff5c01f8d08c16d1419f8d14eabd6)
      ...and 5 more failures.
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\8de7cba9eaf3f8277a748b1a97834f88-006bdbf7-983e-4b71-a777-d0cae873f077) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\8de7cba9eaf3f8277a748b1a97834f88)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1ccbe0288edd62f65586af9879c65973-0d27e140-d63c-4c2f-9bd7-6ebcf454f191) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1ccbe0288edd62f65586af9879c65973)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\cb4e8f2b118d47f35dd7e49671d4b502-460588fd-5219-43e5-888a-4015f68e947c) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\cb4e8f2b118d47f35dd7e49671d4b502)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\a0d121f6bf0a0cd1ab39dd6508a57d7e-9a300cef-f2d9-4cf6-a112-598de9f17a97) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\a0d121f6bf0a0cd1ab39dd6508a57d7e)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\59c8832e15832b6df264a2872e671865-c6fde3de-4dbc-44da-ac9e-e473d692bcf4) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\59c8832e15832b6df264a2872e671865)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\12737c2f4a859b2f660c843c79d45a86-07faf332-d9dd-4535-a3dd-71a93e78888b) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\12737c2f4a859b2f660c843c79d45a86)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\28bed1eec1ad409b80a9bdcf26268253-82bee0f8-c8ed-4a40-ba48-1e027e4ff52c) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\28bed1eec1ad409b80a9bdcf26268253)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1ae2b520ba5c3afbc50058c5642ff12b-5ccb2758-0c79-4e87-8b09-103300f36438) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1ae2b520ba5c3afbc50058c5642ff12b)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\53938ff7056193752b70a38472c61efe-943788c1-3281-48fd-bdb5-2e8a610f36a1) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\53938ff7056193752b70a38472c61efe)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\1fbff5c01f8d08c16d1419f8d14eabd6-6ba1bb7e-2374-419d-b796-026621206032) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\1fbff5c01f8d08c16d1419f8d14eabd6)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\4c981186b4122d945d59729713faa874-eb332220-020c-49a0-a719-6a95ef04fa8a) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\4c981186b4122d945d59729713faa874)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\d3cb33c3cd0aca15fe3af54c682403b6-fa06d19d-2546-425d-afef-718539c6b7ba) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\d3cb33c3cd0aca15fe3af54c682403b6)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\6cc1ea7f8b25deee0eef247b8a70afc7-1d064c3b-b90a-4aef-8d74-eaf34ba5f5e2) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\6cc1ea7f8b25deee0eef247b8a70afc7)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\9e4ecbb3cd2e58cea1d6063482ce2a7f-40ac6764-4211-4323-89f2-4e9ff28d5599) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\9e4ecbb3cd2e58cea1d6063482ce2a7f)
   > Could not move temporary workspace (C:\Users\SONG\.gradle\caches\8.8\transforms\f13b3b2bb7d9cc80066904a5b0828f98-09080a00-737f-431a-9c60-5c162791f51e) to immutable location (C:\Users\SONG\.gradle\caches\8.8\transforms\f13b3b2bb7d9cc80066904a5b0828f98)

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 12s
오후 6:24:13: Execution finished ':classes'.
이런식으로 아에 graddle 이 설정이 안됩니다

Comment (speter6501) [reply gradle/wrapper/gradle-wrapper.properties:3]: 빌드도 안되고요

Comment (speter6501) [reply gradle/wrapper/gradle-wrapper.properties]: 인텔리제이 재설치및 gradle 캐쉬및 폴더삭제

**Review by l2chmnl — APPROVED** [review]
확인했습니다.

**Pull request closed without merging**